<launch>

	<node pkg="turtlesim" type="turtlesim_node" name="sim"/>
	<node pkg="chapter4_tutorials" type="example1" name="example1" />	
	<param name="axis_linear" value="1" type="int" />
	<param name="axis_angular" value="0" type="int" />
	<node respawn="true" pkg="joy"
		type="joy" name="teleopJoy">
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.12" />
	</node>

</launch>
